src/background.ts

```diff
@@ -110,8 +110,13 @@ function turnFilteringOn(): void {
 
 // reloads tab that is currently in focus
 function reloadActive(): void {
-  chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
-    chrome.tabs.reload(tabs[0].id)
+  getStorage().then((storage) => {
```

Comment:
love the use of the helper functions! kind of wondering what cleanDomain does exactly? Does it just get rid of the `https://` and `www.`?

Reply:
yeah just keeps the host (e.g. youtube.com), basically just strips those two you mentioned as well as any sub-path (e.g. youtube.com/channel/whatever)